monitoring/uss_qualifier/suites/uspace/network_identification.yaml

```
@@ -0,0 +1,43 @@
+name: U-Space network identification
+resources:
+  flights_data: resources.netrid.FlightDataResource
+  service_providers: resources.netrid.NetRIDServiceProviders
+  observers: resources.netrid.NetRIDObserversResource
+  evaluation_configuration: resources.netrid.EvaluationConfigurationResource
+  dss_instances: resources.astm.f3411.DSSInstancesResource
+
+actions:
+- test_suite:
+    suite_type: suites.astm.netrid.f3411_22a
+    resources:
+      flights_data: flights_data
+      service_providers: service_providers
+      observers: observers
+      evaluation_configuration: evaluation_configuration
+      dss_instances: dss_instances
+  on_failure: Continue
+participant_verifiable_capabilities:
+- id: uspace_netrid_service_provider
+  name: U-Space ASTM NetRID Service Provider
+  description: Participant fulfills testable requirements necessary to act as a NetRID Service Provider according to F3411-22a with additional information required for U-Space.
```

I think this would simply be "U-space NetRID Service Provider", or if we wanted to get specific, "U-space NetRID Service Provider complying via ASTM F3411-22a".  The use of F3411-22a is a means of compliance for the regulation, but not a/the core of the capability we're verifying.

```suggestion
  name: U-Space NetRID Service Provider
  description: Participant fulfills testable requirements necessary to act as a provider of the mandatory NetRID U-space service via the use of ASTM F3411-22a as the primary means of compliance.
```